i18n/en/docusaurus-plugin-content-docs/current/04-deployment/18Exporter.md

```diff
@@ -1,355 +1,282 @@
+
+   ```java
+    Set<Map.Entry<String, Long>> brokerOffsetEntries = brokerOffsetMap.entrySet();
+           for (Map.Entry<String, Long> brokerOffsetEntry : brokerOffsetEntries) {
+               metricsService.getCollector().addTopicOffsetMetric(clusterName, brokerOffsetEntry.getKey(), topic,
+                   brokerUpdateTimestampMap.get(brokerOffsetEntry.getKey()), brokerOffsetEntry.getValue());
+           }
+       }
+       log.info("topic offset collection task finished...." + (System.currentTimeMillis() - start));
+   }
+   ```
+
+### Rocketmq-exporter collects metrics flowchart 
 
 <img src="https://tva1.sinaimg.cn/large/e6c9d24egy1h4l64nrfwrj20oz0btmze.jpg"></img>
 
-## 快速开始
+## Quick star
 
-### 配置 `application.yml`
+### Configure `application.yml`
 
-`application.yml` 中重要的配置主要有:
+Important configurations in `application.yml` include:
 
-- server.port 设置 promethus 监听 rocketmq-exporter 的端口, 默认为 5557
+- server.port sets the port that Prometheus listens to for the rocketmq-exporter, with a default value of 5557.
 
-- rocketmq.config.webTelemetryPath 配置 promethus 获取指标的路径,默认为 /metrics ，使用默认值即可.
+- rocketmq.config.webTelemetryPath configures the path for Prometheus to obtain metrics, with a default value of /metrics. The default value can be used.
 
-- rocketmq.config.enableACL 如果 RocketMQ 集群开启了 ACL 验证,需要配置为 true, 并在 accessKey 和 secretKey 中配置相应的 ak, sk.
+- rocketmq.config.enableACL If the RocketMQ cluster has enabled ACL verification, it needs to be set to true and the corresponding ak and sk need to be configured in accessKey and secretKey.
 
-- rocketmq.config.outOfTimeSeconds 用于配置存储指标和相应的值的过期时间,若超过该时间,cache 中的 key 对应的节点没有发生写更改,则会进行删除.一般配置为 60s 即可(根据 promethus 获取指标的时间间隔进行合理配置,只要保证过期时间大于等于 promethus 收集指标的时间间隔即可)
+- rocketmq.config.outOfTimeSeconds is used to configure the expiration time of storing metrics and their values. If it exceeds this time and the key in the cache has not undergone a write change, it will be deleted. Generally, it can be configured as 60s (the time interval for Prometheus to obtain metrics should be reasonably configured according to the expiration time, as long as the expiration time is greater than or equal to the time interval for Prometheus to collect metrics).
 
-- task._.cron 配置 exporter 从 broker 拉取指标的定时任务的时间间隔,默认值为"15 0/1 _ \* \* ?" 每分钟的 15s 拉取一次指标.
+- task._.cron configures the time interval for the exporter to pull metrics from the broker through a scheduled task, with a default value of "15 0/1 _ * * ?" which means it will pull metrics every 15s of every minute.
 
-### 启动 exporter 项目
+### Start exporter application
 
-### 按照 promethus 官网配置启动
+### Start prometheus according to the configuration on its official website
 
-配置 promethus 的 static_config: -targets 为 exporter 的启动 IP 和端口,如: localhost:5557
+Configure prometheus's static_config: -targets to the exporter's starting IP and port, such as: localhost:5557.
 
-### 访问 promethus 页面
+### Access the prometheus page
```

Comment:
Better capitalized prometheus into Prometheus, the same goes for others in the doc. Please check other prometheus usages again. Thx.